### PR TITLE
[WIP] Integrating frontend as Canister for DFINITY

### PR DESCRIPTION
**Pull request by JohannesKlauss**

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -27,7 +27,7 @@
   </head>
```
Comment (plitzenberger) [inline public/index.html:27]: Is this file still necessary when using this `ic-injector.js`?

File: webpack.config.js [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+
+/**
+ * Generate a webpack configuration for a canister.
+ */
+function generateWebpackConfigForCanister(name, info) {
+  if (typeof info.frontend !== 'object') {
+    return;
+  }
+
+  generateIcEntryPoint(name, info);
```
Comment (plitzenberger) [inline webpack.config.js:133]: Isn't this just necessary for the frontend canister?

File: public/index.html (hunk 1 above)
Comment (JohannesKlauss) [reply public/index.html:27]: Yes, since it's used in yarn start I think.

File: webpack.config.js (hunk 2 above)
Comment (JohannesKlauss) [reply webpack.config.js:133]: Yes
